portal_partner_select_all/i18n/portal_partner_select_all.pot

```diff
@@ -14,17 +14,18 @@ msgstr ""
 "Plural-Forms: \n"
 
 #. module: portal_partner_select_all
-#: model:ir.ui.view,arch_db:portal_partner_select_all.portal_wizard
+#: model_terms:ir.ui.view,arch_db:portal_partner_select_all.portal_wizard
 msgid "<i class=\"fa fa-warning\"/> You're about to invite to portal all the contacts list below."
```

Comment:
"You're about to invite all contacts listed below to the portal."

Reply:
Changed it, thank you